tests/Language/Haskell/Stylish/Step/Imports/Tests.hs

```diff
@@ -842,3 +845,43 @@ case30 :: Assertion
 case30 = assertSnippet (step Nothing defaultOptions {separateLists = False})
     ["import           Data.Monoid (Monoid (..))"]
     ["import           Data.Monoid (Monoid(..))"]
+
+--------------------------------------------------------------------------------
+case31 :: Assertion
+case31 = assertSnippet (step Nothing defaultOptions {postQualified = True})
+    ["import           Data.Monoid (Monoid (..))"]
+    ["import Data.Monoid (Monoid (..))"]
+
+--------------------------------------------------------------------------------
+case32 :: Assertion
+case32 = assertSnippet (step Nothing defaultOptions {postQualified = True})
+    ["import qualified Data.Monoid as M"]
+    ["import Data.Monoid qualified as M"]
+
+--------------------------------------------------------------------------------
+case33 :: Assertion
+case33 = assertSnippet (step Nothing defaultOptions {postQualified = True})
+    [ "import Data.Default.Class (Default(def))"
+    , "import qualified Data.Aeson as JSON"
+    , "import qualified Data.Aeson as JSON"
+    , "import Control.Monad"
+    , "import Control.Monad"
+    , ""
+    , "import Data.Maybe (Maybe   (Just, Nothing))"
+    , "import qualified Data.Maybe.Extra (Maybe(Just, Nothing))"
+    , ""
+    , "import Data.Foo (Foo (Foo,Bar), Goo(Goo))"
+    , "import Data.Foo (Foo (Foo,Bar))"
+    , "import Data.Set (empty, intersect)"
+    , "import Data.Set (empty, nub)"
+    ]
+    [ "import Control.Monad"
+    , "import Data.Aeson         qualified as JSON"
+    , "import Data.Default.Class (Default (def))"
+    , ""
+    , "import Data.Maybe         (Maybe (Just, Nothing))"
+    , "import Data.Maybe.Extra   (Maybe (Just, Nothing)) qualified"
+    , ""
+    , "import Data.Foo           (Foo (Bar, Foo), Goo (Goo))"
+    , "import Data.Set           (empty, intersect, nub)"
+    ]
```

Comment:
```suggestion
    ]

--------------------------------------------------------------------------------
case34 :: Assertion
case34 = assertSnippet (step Nothing defaultOptions {postQualified = True})
    [ "import qualified Data.Aeson as JSON (Value)"
    ]
    [ "import Data.Aeson qualified as JSON (Value)"
    ]

--------------------------------------------------------------------------------
case35 :: Assertion
case35 = assertSnippet (step Nothing defaultOptions {postQualified = True})
    [ "import Data.Aeson qualified as JSON (Value)"
    ]
    [ "import Data.Aeson qualified as JSON (Value)"
    ]

--------------------------------------------------------------------------------
case36 :: Assertion
case36 = assertSnippet (step Nothing defaultOptions {postQualified = True})
    [ "import qualified Data.Aeson as JSON (Value)"
    , "import qualified Data.Aeson as JSON (encode, decode)"
    ]
    [ "import Data.Aeson qualified as JSON (Value, decode, encode)"
    ]

--------------------------------------------------------------------------------
case37 :: Assertion
case37 = assertSnippet (step Nothing defaultOptions {postQualified = True})
    [ "import Data.Aeson qualified as JSON (Value)"
    , "import Data.Aeson qualified as JSON (encode, decode)"
    ]
    [ "import Data.Aeson qualified as JSON (Value, decode, encode)"
    ]

```

Reply:
@jhmcstanton I've also added these changes as a PR to your branch, in case that's easier to work with